partiql-planner/src/main/kotlin/org/partiql/planner/PartiQLPlannerBuilder.kt

```diff
@@ -41,6 +46,20 @@ public class PartiQLPlannerBuilder {
         this.passes.addAll(passes)
     }
 
+    /**
+     * Java style method for setting the planner to signal mode
+     */
+    public fun signalMode(): PartiQLPlannerBuilder = this.apply {
+        this.flags.add(PlannerFlag.SIGNAL_MODE)
+    }
+
+    /**
+     * Java style method for setting the planner to quite mode
+     */
+    public fun quiteMode(): PartiQLPlannerBuilder = this.apply {
+        this.flags.remove(PlannerFlag.SIGNAL_MODE)
+    }
```

Comment:
Let's only have one method here, as remember we are setting a boolean. The comment should be very clear which is the default. 

Something as straightforward as this,

```kotlin
public fun warningsAsErrors(): PartiQLPlannerBuilder = this.apply {
        this.flags.add(PlannerFlag.SIGNAL_MODE)
 }
```

We could get more sophisticated if we want to generalize, but we have no need for that now. I don't care about the internal names, but signal/quiet is a bit too vague and associated with values/NaN's whereas gcc uses warn/error lingo in regards to the compiler behavior.

I'm referencing `-Werror` https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html

```
-Werror

    Make all warnings into errors.
```